src/BootstrapTable.js

```diff
@@ -142,7 +142,13 @@ class BootstrapTable extends Component {
     const { options, selectRow } = nextProps;
 
     this.store.setData(nextProps.data.slice());
-    let page = options.page || this.state.currPage;
+
+    let page;
+    if (options.page !== 'undefined') {
```

Comment:
you could also say `options.page != null'`. This will compare with `undefined` as well as `null`.

Note: do not use `!==` see [`null`](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/null)